app/screens/channel/channel_nav_bar/channel_nav_bar.js

```diff
@@ -100,7 +100,7 @@ export default class ChannelNavBar extends PureComponent {
         }
 
         let drawerButtonVisible = false;
-        if (!DeviceTypes.IS_TABLET || this.state.isSplitView || !this.state.permanentSidebar) {
+        if (!this.state.permanentSidebar || !this.state.isSplitView) {
```

Comment:
Hmmm this is also affecting ipads right?

Reply:
You're right, they have a "persistent" sidebar already.
Icon removed for iPads via 3ea6b56e73e68ce130f4bfe576dbde22abc58a4d